test_agent/python/testagent.py

```diff
@@ -60,15 +61,21 @@
 logger = logging.getLogger("File:Line# Debugger")
 logger.setLevel(logging.DEBUG)
 
-RESPONSE_URI = UUri(ue_id=1, ue_version_major=1, resource_id=0)
+RESPONSE_URI = UUri(ue_id=random.randrange(0, 0x7FFF), ue_version_major=1, resource_id=random.randrange(0, 0x7FFF))
```

Comment:
No need for this ?

Reply:
This is the default if for whatever reason it isn't set from the initialize_transport flow, so would be superceded, but nice to have a fallback

Reply:
I see, Can you change resource_id range to just equal to 0 ?

Reply:
right, yes

Reply:
done